common/src/lib.rs

```diff
@@ -118,3 +118,22 @@ where
 async fn never_bail() -> Result<bool, Error> {
     Ok(false)
 }
+
+/// A wrapper struct that does nothing other than elide the inner value from
```

Comment:
I wrote https://docs.rs/debug-ignore a bit ago which we use :)

Reply:
The trick is that it must also implement `JsonSchema` for this particular use case, and anything else we ever want to use it in a struct for as well. For that reason it seems to make sense to keep it within Omicron so we can adjust it as needed.

Reply:
sighhhh, yeah.

Reply:
(Could you add a comment about this?)